Modules/_functoolsmodule.c

```diff
@@ -1471,6 +1682,7 @@ static PyType_Spec lru_cache_type_spec = {
 
 /* module level code ********************************************************/
 
+
```

Comment:
Always make sure to not introduce non-related style changes. Also, please read carefully through PEP-7 and apply it to your C code.

```suggestion
```